lib/oc-tests/test_shareFile.py

```diff
@@ -113,8 +113,12 @@ def sharer(step):
 
     step (4,'Sharer shares files')
 
-    user1 = "%s%i"%(config.oc_account_name, 1)
-    user2 = "%s%i"%(config.oc_account_name, 2)
+    if config.use_ldap_users:
```

Comment:
What about using only one config key?
```
if config.ldap_usernames:
    user1 = config.ldap_usernames[0]
    ......
```
If there are ldap usernames, then we use LDAP instead of the DB.